app/_locales/pt_BR/messages.json

```
@@ -3,105 +3,180 @@
   },
   "accountSelectionRequired": {
     "message": "Você precisa selecionar uma conta!"
   },
+  "active": {
+    "message": "Ativo"
+  },
+  "activity": {
+    "message": "Atividade"
+  },
   "activityLog": {
-    "message": "registro de atividade"
+    "message": "log de atividades"
+  },
+  "addAccount": {
+    "message": "Adicionar uma conta"
   },
   "addAcquiredTokens": {
-    "message": "Adicione os tokens que você adquiriu usando o MetaMask"
+    "message": "Adicionar os tokens que você adquiriu usando o MetaMask"
   },
   "addAlias": {
-    "message": "Adicionar pseudônimo"
+    "message": "Adicionar alias"
   },
   "addNetwork": {
-    "message": "Adicionar Rede"
+    "message": "Adicionar rede"
   },
   "addRecipient": {
-    "message": "Adicionar Destinatário"
+    "message": "Adicionar destinatário"
   },
   "addSuggestedTokens": {
-    "message": "Adicionar Tokens Sugeridos"
+    "message": "Adicionar tokens sugeridos"
   },
   "addToAddressBook": {
-    "message": "Adicionar à lista de endereços"
+    "message": "Adicionar ao catálogo de endereços"
   },
   "addToAddressBookModalPlaceholder": {
-    "message": "Exemplo: John D."
+    "message": "p.ex.: John D."
   },
   "addToken": {
     "message": "Adicionar token"
   },
   "addTokens": {
-    "message": "Adicionar Tokens"
+    "message": "Adicionar tokens"
   },
   "advanced": {
-    "message": "Avançado"
+    "message": "Avançadas"
   },
   "advancedOptions": {
     "message": "Opções avançadas"
   },
   "advancedSettingsDescription": {
-    "message": "Acesse recursos do desenvolvedor, baixe registros de estado, redefina a conta, configure testnets e personalize RPC"
+    "message": "Acessar recursos de desenvolvedor, baixar logs, redefinir conta, configurar testnets e RPC personalizadas"
+  },
+  "affirmAgree": {
+    "message": "Concordo"
+  },
+  "aggregatorFeeCost": {
+    "message": "Taxa de rede do agregador"
+  },
+  "alertDisableTooltip": {
+    "message": "Isso pode ser alterado em \"Configurações > Alertas\""
+  },
+  "alertSettingsUnconnectedAccount": {
+    "message": "Navegar por um website com uma conta não conectada selecionada"
+  },
+  "alertSettingsUnconnectedAccountDescription": {
+    "message": "Este alerta é exibido no pop-up quando você estiver navegando por um site conectado da Web3, mas a conta selecionada no momento não estiver conectada."
+  },
+  "alerts": {
+    "message": "Alertas"
+  },
+  "alertsSettingsDescription": {
+    "message": "Habilitar ou desabilitar cada alerta"
+  },
+  "allowExternalExtensionTo": {
+    "message": "Permitir que esta extensão externa:"
+  },
+  "allowOriginSpendToken": {
+    "message": "Permitir que $1 gaste seus(suas) $2?",
+    "description": "$1 is the url of the site and $2 is the symbol of the token they are requesting to spend"
+  },
+  "allowThisSiteTo": {
+    "message": "Permitir que este site:"
+  },
+  "allowWithdrawAndSpend": {
+    "message": "Permitir que $1 saque e gaste do seguinte valor:",
```

I believe the intention was to spend up to a value.

I'd change the translation to:
```
Permitir que $1 saque e gaste um valor máximo de:
```

Or maybe even simpler:
```
Permitir que $1 saque e gaste no máximo:
```

---

Good catch! That does seem more correct. I'll apply the simpler suggestion:
```suggestion
    "message": "Permitir que $1 saque e gaste no máximo:",
```